tests/unit/style/atomic_refcell.rs

```diff
@@ -12,9 +13,67 @@ struct Bar {
     f: Foo,
 }
 
+impl Default for Bar {
+    fn default() -> Self {
+        Bar { f: Foo { u: 42 } }
+    }
+}
+
```

Comment:
Would be good yeah, but I don't quite have the cycles to come up with an elegant way to actually exercise the concurrent cases. I'll add a FIXME.